app/assets/stylesheets/v3/layout_components/forms.scss

```diff
@@ -145,8 +145,8 @@ input[type="checkbox"] + label,
 .checkbox-label {
   display: inline-block;
   width: 85%;
-  margin-top: 10px;
-  margin-bottom: 10px;
+  margin-top: 15px;
+  margin-bottom: 15px;
   margin-left: 55px;
```

Comment:
What do you think about writing all these margins as one declaration?
Also, the px values are one pixel off of the base values, so could we write them as
`margin: $base-margin 0 $base-margin $base-margin*3.5`, if that works for you.